src/main/java/com/cleanroommc/groovyscript/compat/mods/iceandfire/IceAndFire.java

```diff
@@ -0,0 +1,25 @@
+package com.cleanroommc.groovyscript.compat.mods.iceandfire;
+
+import com.cleanroommc.groovyscript.compat.mods.GroovyPropertyContainer;
+import net.minecraftforge.fml.common.Loader;
+
+public class IceAndFire extends GroovyPropertyContainer {
+
+    public final FireForge fireForge = new FireForge();
+    public final IceForge iceForge = new IceForge();
+    public final LightningForge lightningForge;
+
+    public IceAndFire() {
+        lightningForge = isRotN() ? new LightningForge() : null;
+    }
+
+    public static boolean isRotN() {
+        for (var container : Loader.instance().getActiveModList()) {
+            if ("iceandfire".equals(container.getModId())) {
```

Comment:
you can get mod container from a map. Its something like `Loader.instance().getIndexedModList()`